addons/project/static/src/js/tours/project.js

```diff
@@ -122,51 +122,12 @@ registry.category("web_tour.tours").add('project_tour', {
     trigger: "div[role='article']",
     mobile: true,
     run: "click",
-}, {
-    trigger: 'a[name="sub_tasks_page"]',
-    content: _t('Open sub-tasks notebook section'),
-    run: 'click',
-}, {
-    trigger: '.o_field_subtasks_one2many .o_list_renderer a[role="button"]',
-    content: _t('Add a sub-task'),
-    run: 'click',
-}, {
-    trigger: '.o_field_subtasks_one2many div[name="name"] input',
-    content: markup(_t('Give the sub-task a <b>name</b>')),
-    run: 'text New Sub-task'
-}, {
-    trigger: ".o_form_button_save",
-    extra_trigger: '.o_form_project_tasks .o_form_dirty',
-    content: markup(_t("You have unsaved changes - no worries! Odoo will automatically save it as you navigate.<br/> You can discard these changes from here or manually save your task.<br/>Let's save it manually.")),
-    position: "bottom",
```

Comment:
After discussing with Alexandra, we think we should just execute those steps when the tour is automatically launched (in a python test for instance).

Since that tour is launched by a python test, we could just add `auto: true` in the steps you removed, to juat execute those tests when the tour is automatically started. 🙂

Reply:
required changes done